src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
```

Comment:
```suggestion
public class Lottery {
```

class Lottery should not extend ColorSupplier, we should use instanse of ColorSupplier class inside Lottery class

Reply:
what about this comment?